src/Aspire.Hosting.Azure/AzureOpenAIDeploymentResource.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.ApplicationModel;
+
+/// <summary>
+/// Represents an Azure OpenAI Deployment resource.
+/// </summary>
+public class AzureOpenAIDeploymentResource : Resource,
```

Comment:
If we are defining a connection string format, perhaps we should include a connection string that includes the deployment name too?

Reply:
.. no need to block on this feedback though.